comfy_cli/error_codes.py

```
@@ -945,6 +945,62 @@ class ErrorCode:
+    ErrorCode(
+        "distribution_not_enabled",
+        "The builder returned 403 FEATURE_NOT_ENABLED: the developer platform is in limited beta and the "
+        "signed-in account is not enabled for it yet.",
+        "the developer platform is in limited beta; request access, then sign in with an enabled account",
+    ),
+    ErrorCode(
+        "distribution_missing_comfy_version",
+        "`comfy distribution create` was given a definition with no `baseComfyVersion`. The builder can create "
+        "the distribution but cannot cut a build without a pinned ComfyUI version, so create fails fast here "
+        "instead of surfacing a raw builder 400 after doing work. `details.path` carries the definition path.",
+        "re-scan with `--comfy-version <ref>` (a tag, branch, or commit), or add a `baseComfyVersion` to the definition JSON",
```

Pylint (4.0.6)

[convention] Line too long (126/120)

(C0301)